features/configure-clocksource/configure.d/configure-clocksource

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+#==============================================================================
+# Copyright (C) 2017 Stephen F. Norledge and Alces Software Ltd.
+#
+# This file/package is part of Alces Clusterware.
+#
+# Alces Clusterware is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License
+# as published by the Free Software Foundation, either version 3 of
+# the License, or (at your option) any later version.
+#
+# Alces Clusterware is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this package.  If not, see <http://www.gnu.org/licenses/>.
+#
+# For more information on the Alces Clusterware, please visit:
+# https://github.com/alces-software/clusterware
+#==============================================================================
+require clocksource
+require files
+
+main() {
+    local temp_file
+
+    if ! clocksource_is_master_node; then
+        files_load_config config config/cluster
+        temp_file=$(mktemp /tmp/clocksource.XXXXXXXXX)
+        if wget -O $temp_file http://$cw_CLUSTER_master/customizer/cluster-clocksource.rc >/dev/null 2>&1
+            then
+            cp -f $temp_file $cw_ROOT/etc/cluster-clocksource.rc
+            chmod +r $cw_ROOT/etc/cluster-clocksource.rc
+        fi
+        rm -f $temp_file
+    fi
+    
+    if files_load_config --optional cluster-clocksource; then
```

Comment:
This doesn't match the spec.

> If on master:
> - Read ${cw_ROOT}/etc/config/cluster/config.rc and, if cw_CLUSTER_clocksource is set, perform alces configure clocksource using that value, otherwise no-op.

If this code is run on a master node, it will only perform clocksource configuration if `cluster-clocksource.rc` is present, and it will use the value set in that file, rather than the value (if present) in `config/cluster/config.rc` as specced.